scripts/sv/fermiLite/scanBam.sh

```diff
@@ -22,13 +22,15 @@ REF_FASTA="${MASTER_NODE}$5"
 REF_INDEX_IMAGE="$6"
 INTERVAL_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.intervals/')
 KMER_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.kmers/')
+ALTS_KILL_LIST=$(echo "${REF_FASTA}" | sed 's/.fasta$/.kill.alts/')
```

Comment:
Just for the sake of documentation, even though we've already discussed it:  The missing contigs in the alt file stored in the reference directory do not list contigs that do not share substantial homology with one of chromosomal contigs.  Those aren't a problem for the mapper, so they're omitted.
We'll use the list of all non-chromosomal contigs for our SV cross-contig kill list.